tests/unit/test_lidar_parser.py

```diff
@@ -188,6 +187,7 @@ def test_no_oscillation_with_single_reading(
         result, oscillation = lidar_parser_instance.run(lidar_detection_up)
         assert not result
         assert oscillation is None
+        assert lidar_parser_instance.lidar_readings[0].angle == 15.0
```

Comment:
we want to check that the returned oscillation has the angle 15 in it not the instance which is the test case that you created.

Reply:
oscillation not being returned here, as solo angle being provided to our lidar_parser instance. so the last assert basically looks at the leftovers, that the 15 degree ange is being processed, but not returned